pt-BR/mail/chrome/messenger/addressbook/abSelectAddressesDialog.dtd

```diff
@@ -36,32 +36,31 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!-- Title -->
-<!ENTITY selectAddressWindow.title           "Select Addresses">
+<!ENTITY selectAddressWindow.title           "Selecione os endereços">
 
 <!-- Buttons -->
-<!ENTITY toButton.label                      "To->">
+<!ENTITY toButton.label                      "Para->">
 <!ENTITY toButton.accesskey                  "T">
 <!ENTITY ccButton.label                      "Cc->">
 <!ENTITY ccButton.accesskey                  "C">
-<!ENTITY bccButton.label                     "Bcc->">
+<!ENTITY bccButton.label                     "Cco->">
```

Comment:
I saw somewhere in the dtd files that there was a instruction to not translate Bcc, but on that same file it was already translated in the previous version of Postbox. For consistency, I also had to translate. I'll ask Sherman if I should change all of them retroactively, but Cco is what we use in PTBR.